unsafe/src/main/scala/monocle/unsafe/MapTraversal.scala

```diff
@@ -15,6 +15,18 @@ import scala.collection.immutable.Map
 object MapTraversal {
   implicit def mapEach[K, V]: Each[Map[K, V], V] = fromTraverse[Map[K, ?], V]
 
+  implicit def mapTraversal[K, V]: Traversal[Map[K, V], (K, V)] =
```

Comment:
optics (`Iso`, `Lens`, ...) are not unique per type, so we don't define them with implicit as opposed to typeclass instance.

Reply:
maybe we should use another name like `mapKeyValues`, `allKeyValues`, naming is hard ^^

Reply:
Thanks. I removed the implicit and changed the name to `allKeyValues`.